pytest_plugins/issue_handlers.py

```diff
@@ -42,14 +66,39 @@ def pytest_collection_modifyitems(session, items, config):
 
     pytest.issue_data = generate_issue_collection(items, config)
 
-    # No item deselect, just adding skipif markers
+    # Add skipif markers, and modify collection if there were BZ options passed
```

Comment:
s/passed/provided ?

Or


```suggestion
    # Add skipif markers, and modify collection based on BZ option.
```